system/Images/Exceptions/ImageException.php

```diff
@@ -16,52 +16,52 @@
 
 class ImageException extends FrameworkException implements ExceptionInterface
 {
-    public static function forMissingImage()
+    public static function forMissingImage(): self
```

Comment:
This is a breaking change.
https://3v4l.org/GGS0f
Please use doc comment.

Reply:
Done.

Reply:
Done.